jsk_fetch_robot/fetcheus/CMakeLists.txt

```diff
@@ -7,7 +7,12 @@ if($ENV{ROS_DISTRO} STREQUAL "hydro")
   return()
 endif()
 
-find_package(catkin REQUIRED fetch_description)
+find_package(catkin REQUIRED
+    collada_urdf
+    euscollada
```

Comment:
If cmake does not use variables defined in collada_urdf and euscollada, we do not need that.


I think roseus is requred to compile fetch_driver_msgs, but it from kinetc, you do not need that, so please add some comment like `;; <= indigo required this,`

Reply:
@k-okada Fixed. Please correct me if I got it wrong.